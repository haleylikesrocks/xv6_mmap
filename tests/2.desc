Anonymous mmap Test : Testing Anonymous mmap with PROT_WRITE flag
