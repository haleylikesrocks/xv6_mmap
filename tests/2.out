XV6_TEST_OUTPUT : mmap good
XV6_TEST_OUTPUT : Ret of memcmp 0
XV6_TEST_OUTPUT : mmap() has correctly cleared the newly mapped anonymous memory region
XV6_TEST_OUTPUT : munmap good
